runner/config.go

```diff
@@ -327,6 +329,9 @@ func (c *Config) preprocess() error {
 	// CMD will not recognize relative path like ./tmp/server
 	c.Build.Bin, err = filepath.Abs(c.Build.Bin)
 
+	// Account for spaces in filepath
+	c.Build.Bin = fmt.Sprintf("%q", c.Build.Bin)
```

Comment:
I think this is causing an error on Windows, the exe can't be found and the path includes escape characters like this:

```
'\"c:\\go\\hello\\tmp\\hello.exe\"' is not recognized as an internal or external command,
operable program or batch file.
```

When if it was actually missing the error would be: this when the exe is missing and I run it from a command prompt
```
'c:\go\hello\tmp\hello.exe' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
yep, that is not working properly